valuesets/country-2-codes.json

```diff
@@ -1,1749 +1,1749 @@
 {
   "valueSetId": "country-2-codes",
-  "valueSetDate": "2021-06-03",
+  "valueSetDate": "2019-11-01",
   "valueSetValues": {
     "AD": {
       "display": "Andorra",
       "lang": "en",
       "active": true,
       "system": "urn:iso:std:iso:3166",
-      "valueSetId": "iso3166-1-2"
+      "version": "1.0"
```

Comment:
Standardize on 3-digit [semver](https://semver.org) - so this should be 1.0.0

Reply:
Value set date (initial) was now taken from the original source - FHIR. IT should be updated after any manual change of the value set. As regards new codes like WHO, UNHCR - those should be proposed and agreed by the Semantic Sg.
If we see a business need for those two codes, we need to formulate a CP (well at least a clarification "why" and "what" we propose to add). 
As regard version for value set entries, it is expressed according to the conventions used by a code system where taken from. I don't think that we have to change it. As regards iso-3166, no version information is provided, as we don't have access to the paid version. So, I removed version information from the value set.

Reply:
I don't want to be too formal here, but we should in writing:
1. Specify the business need, i.e. why (in which situation) we want to enable special entries in the value set.
2. Specify which entries we want to add, i.e. code, display label, code system and version information.
This we should send to SgS for acceptance.